stdlib/public/core/Array.swift

```diff
@@ -255,21 +255,32 @@
 /// Foundation type.
 ///
 /// The following example shows how you can bridge an `Array` instance to
-/// `NSArray` to use the `write(to:atomically:)` method. In this example, the
+/// `NSArray` to use the `write(to:)` method. In this example, the
 /// `colors` array can be bridged to `NSArray` because the `colors` array's
-/// `String` elements bridge to `NSString`. The compiler prevents bridging the
-/// `moreColors` array, on the other hand, because its `Element` type is
-/// `Optional<String>`, which does *not* bridge to a Foundation type.
+/// `String` elements bridge to `NSString`. Despite `Optional<String>` does not
+///  bridge to Foundation type, the array which elemets are of `Optional<String>`
+///  type does bridge. `.some(String)` values are bridges as `NSString` and `nil`
+///  values are bridged as `NSNull`. Therefore `moreColors` could be bridged
+///  to `NSArray` as well. Because `write(to:)` method throws exception if it
+///  encounters `NSNull` values, `do catch` should be used to check if write
+///  succeeded.
```

Comment:
The behavior of `write(to:)` belongs on that symbol's reference, we don't need to repeat it here.